defining_policies_and_profiles/topics/editing_policy_event_assignments.adoc

```diff
@@ -8,7 +8,7 @@ The policy evaluates its scopes and conditions when specified events occur in yo
 
 . Click image:../images/1847.png[image] (*Configuration*), image:../images/1880.png[image] (*Edit this Policy's Event assignments*).
 
-. Check all the events you want to assign to this policy. For a description of the events, see link:https://access.redhat.com/documentation/en/red-hat-{product-title}/version-4.1-beta/defining-policies-and-profiles/#chap_events[Events].
+. Check all the events you want to assign to this policy. For a description of the events, see link:https://access.redhat.com/documentation/en/{product-title}-{product-title}/version-{product-version}-beta/defining-policies-and-profiles/#chap_events[Events].
```

Comment:
This is another one where we can't do a simple search / replace. Instead, use conditional text, like 
The documentation is at 
ifdef::manageiq[]
http://whatever
endif[]
ifdef::cfme[]
http://access.redhat.com/blah
endif[]